google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddPrices.java

```diff
@@ -0,0 +1,275 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.extensions;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v2.common.AdScheduleInfo;
+import com.google.ads.googleads.v2.common.Money;
+import com.google.ads.googleads.v2.common.PriceFeedItem;
+import com.google.ads.googleads.v2.common.PriceOffer;
+import com.google.ads.googleads.v2.enums.DayOfWeekEnum.DayOfWeek;
+import com.google.ads.googleads.v2.enums.ExtensionTypeEnum.ExtensionType;
+import com.google.ads.googleads.v2.enums.MinuteOfHourEnum.MinuteOfHour;
+import com.google.ads.googleads.v2.enums.PriceExtensionPriceQualifierEnum.PriceExtensionPriceQualifier;
+import com.google.ads.googleads.v2.enums.PriceExtensionPriceUnitEnum.PriceExtensionPriceUnit;
+import com.google.ads.googleads.v2.enums.PriceExtensionTypeEnum.PriceExtensionType;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.CustomerExtensionSetting;
+import com.google.ads.googleads.v2.resources.ExtensionFeedItem;
+import com.google.ads.googleads.v2.services.CustomerExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.CustomerExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemOperation;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemServiceClient;
+import com.google.ads.googleads.v2.services.MutateCustomerExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateExtensionFeedItemsResponse;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Int32Value;
+import com.google.protobuf.Int64Value;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Adds a price extension and associates it with an account. Campaign targeting is also set using
+ * the specified campaign ID. To get campaigns, run GetCampaigns.
```

Comment:
Missing **.java** after "GetCampaigns"

Reply:
I don't think this is necessary. It's pretty clear from the context that this means GetCampaigns.java.

Reply:
I agree this is nit. I brought the point because I saw it done this way in other code examples, would be great to make it consistent.